src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1306,12 +1309,29 @@ static String getReferenceBases(final Allele refAllele, final Allele altAllele,
                     reference.getWindow().getStart() - referenceWindow,
                     reference.getWindow().getEnd() + referenceWindow + indelAdjustment);
 
+        // Get the reference bases for this interval.
+        // This grossness is required for now as a fix for the hg19 vs b37 reference contig name issue:
+        byte[] referenceBases;
+        try {
+            referenceBases = reference.getBases(refBasesInterval);
+        }
+        catch (final UserException.MissingContigInSequenceDictionary | UserException.NoDataAtRequestedContig ex) {
```

Comment:
We should never include throwing exceptions, except in erroneous code paths.  This goes double for code that will be run per variant.
Detect whether there will be a missing contig and then use an `if` statement, please.  Or some similar approach.

Reply:
Totally obviated via changes discussed in person.